test/sslapitest.c

```diff
@@ -3763,9 +3774,9 @@ static int test_ciphersuite_change(void)
 # if !defined(OPENSSL_NO_CHACHA) && !defined(OPENSSL_NO_POLY1305)
```

Comment:
These ifdef's are no longer relevant if CHACHA is removed here..

Reply:
Fixed